python_course_1604/class_01/exercize_06_while_break_else.py

```diff
@@ -38,11 +38,11 @@ def count_until(match_value):
 
     Please use while-break-else construct.
     """
+    i = 0
     while has_next():
         temp = get_next()
-        i = 0
         if temp == match_value:
-            i = i + 1
+            i += 1
```

Comment:
Pleas read better specifications. These are not well implemented:

- It loops until get_next() returns a value equal to match_value
- It returns the number of times it calls get_next().
- It raises KeyError when has_next() returns False before searched value is found.